snapchat_dl/snapchat_dl.py

```diff
@@ -125,11 +129,18 @@ def download(self, username):
                     dump_response(media_json, filename_json)
 
                 media_output = os.path.join(dir_name, filename)
+                
+                if os.path.isfile(media_output):
+                    stories_already_exists +=1
+                    continue
+
```

Comment:
`download_url` already handles case when file is downloaded, remove `continue` here and imply increment variable

Reply:
@Walmann please format this using `black`